packages/context/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.common.json",
+  "compilerOptions": {
+    "outDir": "./lib"
```

Comment:
`tsconfig.common.json` specifies `target:es6` ([source](https://github.com/strongloop/loopback-next/blob/f032292e1aa048b713a28b02dede0e951f4af84c/tsconfig.common.json#L12)). I think we should change the output dir to `lib6` here, to prevent the situation where `npm run build` is placing es2017 files into `lib` and VSCode tooling places `es2015` files into the same directory.